Mathlib/LinearAlgebra/ExteriorAlgebra/Grading.lean

```diff
@@ -61,6 +61,7 @@ def GradedAlgebra.liftι :
   lift R ⟨by apply GradedAlgebra.ι R M, GradedAlgebra.ι_sq_zero R M⟩
 #align exterior_algebra.graded_algebra.lift_ι ExteriorAlgebra.GradedAlgebra.liftι
 
+set_option maxHeartbeats 400000 in
```

Comment:
Can you confirm this is still needed? This PR has had a lot of churn since you started!

Reply:
In a local test neither of these two lines are needed anymore (also not the following line). `count_heartbeats in` reports
> Used 144125 heartbeats, which is less than the current maximum of 200000.

Reply:
Actually, looks like we can remove all 4 `set_option`s regarding heartbeats in this file now :tada:

Reply:
```suggestion
```

Reply:
Let's just remove the new one in this PR; I suspect the old ones are already unnecessary on master.